src/com/core/tictactoe/GamesController.java

```diff
@@ -0,0 +1,38 @@
+package com.core.tictactoe;
+
+public class GamesController {
+
+    CommandLineUI commandLineUI;
+    Game game;
+
+    GamesController(CommandLineUI commandLineUI) {
+        this.commandLineUI = commandLineUI;
+        this.game = null;
+    }
+
+    void gamesMenu() {
+        boolean loopThroughOptions = true;
+        while (loopThroughOptions) {
+            this.commandLineUI.gamingMenu();
+            loopThroughOptions = this.actOnOption(this.getUserOption());
+        }
+    }
+
+    private String getUserOption() {
+        return this.commandLineUI.getUserInput();
+    }
+
+    private boolean actOnOption(String pickedOption) {
+        if (pickedOption.matches("[1]")) {
+            this.playANewGame();
+        } else return !pickedOption.matches("[2]");
+        return true;
+    }
+
+    private void playANewGame() {
+        Game newGame = new Game(this.commandLineUI);
+        this.game = newGame;
```

Comment:
do you need to story game in an instance variable?  If not you could lose this line

Reply:
Hi Sarah! This line is very helpful from testing point of view - thanks to storing a game, I can easily check if there was one played in the previous round and was what its final score.